samples/javascript_nodejs/23.facebook-events/index.js

```diff
@@ -1,92 +1,66 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-const { BotFrameworkAdapter, ConversationState, MemoryStorage, UserState } = require('botbuilder');
-const { BotConfiguration } = require('botframework-config');
+// index.js is used to setup and configure your bot
+
+// Import required packages
 const path = require('path');
 const restify = require('restify');
-const FacebookEventsBot = require('./bot');
-
-// Read botFilePath and botFileSecret from .env file.
-// Note: Ensure you have a .env file and include botFilePath and botFileSecret.
-const ENV_FILE = path.join(__dirname, '.env');
-require('dotenv').config({ path: ENV_FILE });
-
-// Create HTTP server.
-let server = restify.createServer();
-server.listen(process.env.port || process.env.PORT || 3978, function() {
-    console.log(`\n${ server.name } listening to ${ server.url }.`);
-    console.log(`\nGet Bot Framework Emulator: https://aka.ms/botframework-emulator.`);
-    console.log(`\nTo talk to your bot, open facebook-events.bot file in the Emulator.`);
-});
+const { FacebookBot } = require('./bots/facebookBot');
+// Import required bot services. See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const { BotFrameworkAdapter, MemoryStorage } = require('botbuilder');
 
-// .bot file path.
-const BOT_FILE = path.join(__dirname, (process.env.botFilePath || ''));
+// This bot's main dialog.
 
-let botConfig;
-try {
-    // Read bot configuration from .bot file.
-    botConfig = BotConfiguration.loadSync(BOT_FILE, process.env.botFileSecret);
-} catch (err) {
-    console.error(`\nError reading bot file. Please ensure you have valid botFilePath and botFileSecret set for your environment.`);
-    console.error(`\n - The botFileSecret is available under appsettings for your Azure Bot Service bot.`);
-    console.error(`\n - If you are running this bot locally, consider adding a .env file with botFilePath and botFileSecret.\n\n`);
-    process.exit();
-}
 
-const DEV_ENVIRONMENT = 'development';
 
-// Bot name as defined in .bot file
-// See https://aka.ms/about-bot-file to learn more about .bot file its use and bot configuration.
-const BOT_CONFIGURATION = (process.env.NODE_ENV || DEV_ENVIRONMENT);
-
-// Get bot endpoint configuration by service name.
-const endpointConfig = botConfig.findServiceByNameOrId(BOT_CONFIGURATION);
+// Note: Ensure you have a .env file and include LuisAppId, LuisAPIKey and LuisAPIHostName.
```

Comment:
"LuisAppId, LuisAPIKey and LuisAPIHostName." <-- this bot does not use LUIS, so this comment can be removed (or changed to instruct the user to add the MicrosoftAppId and MicrosoftAppPassword to .env)

Reply:
Removed the comment and added "Note: Ensure that you have a MicrosoftAppId and MicrosoftAppPassword added to the .env file."